### PR TITLE
Add exceptions to analysis tool

### PR DESCRIPTION
**Pull request by svilenvul**
See #3995

Signed-off-by: Svilen Valkanov <svilen.valkanov@musala.com>

### CODE REVIEW

File: bundles/model/org.eclipse.smarthome.model.script/src/org/eclipse/smarthome/model/script/actions/LogAction.java [hunk 1]
```diff
@@ -7,6 +7,7 @@
  */
 package org.eclipse.smarthome.model.script.actions;
 
+import org.quartz.utils.FindbugsSuppressWarnings;
```
Comment (kaikreuzer) [inline bundles/model/org.eclipse.smarthome.model.script/src/org/eclipse/smarthome/model/script/actions/LogAction.java]: `org.quartz.utils`? Really?

Comment (svilenvul) [inline bundles/model/org.eclipse.smarthome.model.script/src/org/eclipse/smarthome/model/script/actions/LogAction.java]: [edu.umd.cs.findbugs.annotations.SuppressFBWarnings](http://findbugs.sourceforge.net/api/edu/umd/cs/findbugs/annotations/SuppressFBWarnings.html) would be the best choice, but it is not part of the Target Platform at the moment.

I was also surprised that the code above is working, but it is.

Comment (svilenvul) [inline bundles/model/org.eclipse.smarthome.model.script/src/org/eclipse/smarthome/model/script/actions/LogAction.java]: OK, I found them in the `com.eclipsesource.jaxrs.swagger-all.source_1.5.7.jar`, I will update the PR

Comment (kaikreuzer) [inline bundles/model/org.eclipse.smarthome.model.script/src/org/eclipse/smarthome/model/script/actions/LogAction.java]: `org.quartz` soon won't be part of the target platform anymore, that's why I am asking :-)

Comment (svilenvul) [inline bundles/model/org.eclipse.smarthome.model.script/src/org/eclipse/smarthome/model/script/actions/LogAction.java]: Nope, they are not exported.

Comment (svilenvul) [inline bundles/model/org.eclipse.smarthome.model.script/src/org/eclipse/smarthome/model/script/actions/LogAction.java]: Can we add the spotbugs eclipse plugin to the target platform? It exports the annotations package.

**Comment by svilenvul** [comment]
Will be moved to xml filter file that will be part of #3995

**Pull request closed without merging**